third_party/repositories/scala_2_12.bzl

```diff
@@ -77,6 +77,13 @@ artifacts = {
             "@io_bazel_rules_scala_scala_library",
         ],
     },
+    "org_scalameta_semanticdb_scalac": {
+        "artifact": "org.scalameta:semanticdb-scalac_%s:4.8.4" % scala_version,
```

Comment:
Any reason, why 2.11 has slightly higher version than both 2.12 and 2.13?

Reply:
That is last semanticdb version that supported 2.11. Semanticdb stopped supporting Scala 2.11 in semanticdb 4.8.5. (see https://mvnrepository.com/artifact/org.scalameta/semanticdb-scalac)